packages/utils/src/getCountryCode.js

```diff
@@ -4,19 +4,20 @@
  */
 import { ImportValidationError } from './errors';
 
-export const getCountryCode = (countryName, entityObjects = []) => {
-  if (countryName === 'Demo Land') {
-    return 'DL';
-  }
+// We sometimes use non-official names for countries. The most obvious example is Demo Land, but
+// also Laos instead of Laos People's Democratic Republic, etc.
+const UNOFFICIAL_NAME_TO_CODE = {
+  'demo land': 'DL',
+  laos: 'LA',
+  "côte d'ivoire": 'CI',
+  'federated states of micronesia': 'FM',
+  'pitcairn islands': 'PI',
+};
 
+export const getCountryCode = countryName => {
   const { getCode: getCountryIsoCode } = require('countrynames');
-  // Use the country ISO code if there's a direct match,
-  //otherwise base on the two letter facility code prefix
   const countryCode =
-    getCountryIsoCode(countryName) || entityObjects.length
-      ? entityObjects[0].code.substring(0, 2)
-      : null;
```

Comment:
The problem here was that I'm guessing this code intended to act as a lazily evaluated conditional with an early return (i.e., if getCountryIsoCode returned something, use that), but instead the whole || was just used as the condition for the ternary operator. That results in "if there's a valid ISO code for this country name, use the first two letters of the first entry in the spreadsheet", which is not what anyone would have wanted, I'm sure.

Reply:
Great explanation!